src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -430,7 +430,7 @@ public String getAudioCodec() {
 		if (is3GA()) {
 			return "3GA";
 		} else if (isAACLC()) {
-			return "AAC";
+			return "AAC-LC";
 		} else if (isAC3()) {
```

Comment:
You are surely right.
How `FormatConfiguration `should be used ?
Do you mean something like that `return FormatConfiguration.AAC` ?